standardise_ip.py

```diff
@@ -237,18 +237,21 @@ def processed_cell_values():
 
             yield (row_id, rawvalue, cidrized)
 
+    dictionary_of_rows_ips = {}
     for row_id, rawvalue, cidrized in processed_cell_values():
-        sheet.cell(column=columnnumber+1, row=row_id).value = str(cidrized)
+        #sheet.cell(column=columnnumber+1, row=row_id).value = str(cidrized)
```

Comment:
@pancakewaffles it's better not to commit commented-out lines as it makes the code messy - you can always recover the line from git's history if needed